### PR TITLE
[Merged by Bors] - Add mouse grab example

### LINKED ISSUE #4094: Add example for grabbing the mouse
## How can Bevy's documentation be improved?

Demonstrate [how to grab the mouse / lock the cursor] in a small example.

### PR DESCRIPTION
**Pull request by emersonmx**
Fixes #4094

### CODE REVIEW

File: examples/README.md
Comment (alice-i-cecile) [inline examples/README.md:206]:
```suggestion
`mouse_grab` | [`input/mouse_grab.rs`](./input/mouse_grab.rs) | Demonstrates how to grab the mouse, locking the cursor to the app's screen
```

**Review by alice-i-cecile — CHANGES_REQUESTED** [review]
Looks good! Just a tiny suggestion to improve the description.

**Review by alice-i-cecile — APPROVED** [review]

File: examples/input/mouse_grab.rs
Comment (james7132) [inline examples/input/mouse_grab.rs:10]:
```suggestion
// This system grabs the mouse when you the left mouse button is pressed and releases
```

File: examples/input/mouse_grab.rs
Comment (james7132) [inline examples/input/mouse_grab.rs:11]:
```suggestion
// when the escape key is pressed:
```

**Review by james7132 — CHANGES_REQUESTED** [review]

File: examples/input/mouse_grab.rs
Comment (alice-i-cecile) [reply examples/input/mouse_grab.rs:10]:
```suggestion
// This system grabs the mouse when the left mouse button is pressed
// and releases it when the escape key is pressed
```

IMO splitting this across two lines to match the sentence structure is clearer.

**Comment by alice-i-cecile** [comment]
Awesome. Once I have a second approval I'm ready to merge this :)

**Review by james7132 — APPROVED** [review]

**Pull request closed without merging**
